manifests/quick-start/minimal/kustomization.yaml

```diff
@@ -3,3 +3,7 @@ kind: Kustomization
 
 resources:
   - ../base
+
+patchesStrategicMerge:
+  - overlays/workflow-controller-configmap.yaml
```

Comment:
don't we already have this is quick start somewhere else?

Reply:
We have this in the postgress and the mysql. I am not sure where we like to have it to be honest, should we drop it here?